CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v1.1.4 - 2022-06-16
 
+### Adds
```

Comment:
This probably needs to be the start of a `v1.1.5` section right? Rather than falling under `v1.1.4`?

Reply:
I think we haven't formally released 1.1.4 yet, main is on 1.1.3, so was thinking both could be part of the `v1.1.4` release.

Reply:
Right you are. I was misled by the presence of a release date for 1.1.4 in the changelog. :-)

Reply:
Ack. We normally include dates in the CHANGELOG.md correct? It was that there was a previous date we updated that misled?

Reply:
Right, normally we only add a date for a given release when we actually release it :)